include/rocksdb/options.h

```diff
@@ -847,6 +847,18 @@ struct DBOptions {
   //
   // Dynamically changeable through SetDBOptions() API.
   bool avoid_flush_during_shutdown = false;
+
+  // If enabled it uses two queues for writes, one for the ones with
+  // disable_memtable and one for the ones that also write to memtable. This
+  // allows the memtable writes not to lag behind other writes. It can be used
+  // to optimize MySQL 2PC in which only the commits, which are serial, write to
+  // memtable.
+  bool separate_queue_for_memtable_writes = false;
+
+  // If true WAL is not flushed automatically after each write. Instead it
+  // relies on manual invocation of FlushWAL to write the WAL buffer to its
+  // file.
+  bool manual_wal_flush = false;
```

Comment:
Then we would also use two write queues for this user, which is irrelevant to WAL buffer feature that they intended to have.
I do think the WAL buffer is something that will stay in the code base for long time. The other feature is temporary and will be removed entirely once we push the commits to an earlier stage.